<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bookmark_Dialog</class>
 <widget class="QDialog" name="Bookmark_Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookmark manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/main_window.qrc">
    <normaloff>:/icons/elementary3-icon-theme/apps/64/office-database.svg</normaloff>:/icons/elementary3-icon-theme/apps/64/office-database.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="bookmark_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="autoScrollMargin">
        <number>9</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="verticalWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="vertical_spacer_top">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="page_image_label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource/main_window.qrc">:/icons/notCover.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="page_preview_label">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Page Preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="vertical_spacer_botton">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="line_edit_path"/>
   </item>
   <item>
    <layout class="QGridLayout" name="grid_layout">
     <item row="1" column="3">
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/main_window.qrc">
         <normaloff>:/icons/elementary3-icon-theme/actions/48/dialog-apply.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/dialog-apply.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_remove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/main_window.qrc">
         <normaloff>:/icons/elementary3-icon-theme/actions/64/list-remove.svg</normaloff>:/icons/elementary3-icon-theme/actions/64/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button_exit">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/main_window.qrc">
         <normaloff>:/icons/elementary3-icon-theme/actions/48/dialog-cancel.svg</normaloff>:/icons/elementary3-icon-theme/actions/48/dialog-cancel.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalFrame</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>page_image_label</zorder>
  <zorder>page_preview_label</zorder>
  <zorder>line_edit_path</zorder>
 </widget>
 <resources>
  <include location="../resource/main_window.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_exit</sender>
   <signal>clicked()</signal>
   <receiver>Bookmark_Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
